### PR TITLE
Tweaked notebook and ran linter

### PR DESCRIPTION
**Pull request by ivanmkc**
- Fixed text about bucket names
- Exempted from execution: gcloud command that would cause a kernel failure
- Fixed parameter substitution for REGION and PROJECT_ID.

### CODE REVIEW

**Review by thedriftofwords — APPROVED** [review]

File: .cloud-build/ExecuteChangedNotebooks.py [hunk 1]
```diff
@@ -52,9 +53,12 @@ def run_changed_notebooks(
     Variables are also injected into the notebooks such as the variable_project_id and variable_region.
     """
     test_folders = []
+    test_notebooks = []
```
Comment (ivanmkc) [inline .cloud-build/ExecuteChangedNotebooks.py:56]: These changes are just to support testing for single notebooks.

File: .kokoro/notebooks/test_folders.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 ai-platform-unified/notebooks/official
-ai-platform/notebooks/official
\ No newline at end of file
+ai-platform/notebooks/official
+ai-platform-unified/notebooks/notebook_template.ipynb
```
Comment (ivanmkc) [inline .kokoro/notebooks/test_folders.txt:3]: Add the notebook testing to execution tests.

**Pull request merged**